x/emissions/module/abci.go

```diff
@@ -75,7 +75,17 @@ func EndBlocker(ctx context.Context, am AppModule) error {
 				// No nonces to fulfill
 				continue
 			} else {
+				topic, err := am.keeper.GetTopic(sdkCtx, topicId)
+				if err != nil {
+					sdkCtx.Logger().Warn(fmt.Sprintf("Error getting topic: %s", err.Error()))
+					continue
+				}
 				for _, nonce := range nonces.Nonces {
+					// If the nonce plus the worker submission window is greater than the current block height, then the window is still open => skip
+					if nonce.BlockHeight+topic.WorkerSubmissionWindow > blockHeight {
```

Comment:
Great catch. 

Actually the function that was done to check that cadence `keeper.CheckWorkerCloseCadence` is not being used now. 
[code](https://github.com/allora-network/allora-chain/blob/334fbad25389f1a33553f7b11f72e907ed79bf87/x/emissions/keeper/keeper.go#L2668)

I find to have a function with a name like this signals the intention in a clearer way and helps readability.

Note: its companions `CheckReputerCloseCadence` , `CheckWorkerOpenCadence` are not used either now. 
If they are not used then they can be dropped.

Reply:
cleaned in both places mentioned